neptune/new/exceptions.py

```diff
@@ -1131,3 +1131,26 @@ def __init__(self, missing_feature):
 """
         self.message = message.format(missing_feature=missing_feature, **STYLES)
         super().__init__(message)
+
+
+class NeptuneExperimentCreationConflict(NeptuneException):
+    pass
+
+
+class NeptuneKeyAlreadyExistsError(NeptuneExperimentCreationConflict):
+    def __init__(self, model_key, models_tab_url):
+        message = """
+{h1}
+----NeptuneKeyAlreadyExistsError---------------------------------------------------
```

Comment:
project also has a key (and it also has to be unique) -- so combination of `NeptuneKeyAlreadyExistsError` (without saying that it's about model) and `model_key` parameter isn't good IMO

Reply:
True